api/tests/routes/native/v1/account_test.py

```diff
@@ -1897,3 +1897,32 @@ def assert_code_and_not_active(self, response, user, code):
 
         db.session.refresh(user)
         assert not user.isActive
+
+
+class SuspensionTokenValidationTest:
+    def test_error_when_token_is_invalid(self, client):
+        response = client.get("/native/v1/account/suspend/token_validation/abc")
+
+        assert response.status_code == 400
+        assert response.json["reason"] == "Le token est invalide."
+
+    def test_error_when_token_has_invalid_signature(self, client):
+        token = jwt.encode(
+            {"userId": 1},
+            "fake_jwt_secret_key",
```

Comment:
(détail / question de goût) "fake" pourrait vouloir dire qu'on fournit ici une clé quelconque, comme c'est le cas lorsque cela n'a pas d'impact sur le test (et pour le simplifier). Je propose plutôt d'inclure le mot "wrong" (ou "incorrect") : 

```suggestion
            "wrong_jwt_secret_key",
```